### ISSUE TITLE
Saving training losses to csv

### ISSUE THREAD

**Issue by SebiChesh** [sintel-dev/Orion#413]
Orion version: 0.4.1
Python version: 3.7
Operating System: Ubuntu 20.

### Description

I have been looking through the documentation and I can't seem to find out how to access the loss values for every epoch during training so that I can save them to a CSV (im specifically using TadGAN but will be using other models later on as well). Is this currently a feature? or do you reccomend I fork your library and edit the source code and install manually?

**Comment by sarahmish** [comment]
Hi @SebiChesh, thank you for using Orion!

At the moment we do not have this option available. This would be a great feature to contribute! :)
